content/en/docs/contribute/participating.md

```diff
@@ -32,65 +32,61 @@ aspects of Kubernetes -- the Kubernetes website and documentation.
 
 {{% capture body %}}
 
-## Roles and responsibilities
+## Roles overview
 
-When a pull request is merged to the branch used to publish content (currently
-`master`), that content is published and available to the world. To ensure that
-the quality of our published content is high, we limit merging pull requests to
-SIG Docs approvers. Here's how it works.
+- **Anyone** can contribute to Kubernetes documentation. To contribute, you must [sign the CLA](/docs/contribute/start#sign-the-cla) and have a GitHub account.
+- **Members** of the Kubernetes organization are contributors who have spent time and effort on some area of the Kubernetes project, usually in the form of opening pull requests. See [Community membership](https://github.com/kubernetes/community/blob/master/community-membership.md) for membership criteria.
```

Comment:
It's good to be clear that merely opening pull requests isn't a path to membership.

```suggestion
- **Members** of the Kubernetes organization are contributors who have spent time and effort on the Kubernetes project, usually by opening pull requests with accepted changes. See [Community membership](https://github.com/kubernetes/community/blob/master/community-membership.md) for membership criteria.
```